backend/app/routes/UserController.js

```diff
@@ -229,14 +229,27 @@ router.put("/:id", async (req, res) => {
     if (queriedUser.isVolunteer === true)
       return res.status(400).send("User is already a volunteer\n");
 
+    if (req.body.certificateCode === undefined)
+      return res.status(400).send("Certificate code is required\n");
```

Comment:
ma nel caso in cui uno volesse TOGLIERE il volunteer, perchè deve dare anche il suo codice???

Reply:
è comunque legato allo status di volontario, piuttosto aggiungerei il fatto che quando il campo isVolunteer viene settato a false allora il certificateCode viene settato a undefined.